components/eam/src/physics/cam/physics_types.F90

```diff
@@ -645,6 +648,8 @@ subroutine physics_state_check(state, name)
     ! 2-D variables (at midpoints)
     call shr_assert_in_domain(state%t(:ncol,:),         lt=posinf_r8, gt=0._r8, &
          varname="state%t",         msg=msg)
+    call shr_assert_in_domain(state%pv(:ncol,:),        lt=posinf_r8, gt=0._r8, &
```

Comment:
Do you expect PV to ever be negative? This asserts that PV is always non-negative, and if ran with `state_debug_checks=.true.`, will abort the run if PV is ever even slightly negative.

Reply:
PV will generally be negative anywhere in the southern hemisphere. This is now fixed, with the lower bound updated to negative inf